Directory.Build.targets

```diff
@@ -13,4 +13,9 @@
       <InformationalVersion>$(Version); git-rev-head:$(GitCommit); git-branch:$(GitBranch)</InformationalVersion>
     </PropertyGroup>
   </Target>
+
+  <Import
```

Comment:
Question: should this Import be *last*, as it is here? Or *first*?

Reply:
It needs to be last - I had it first before and it didn't work. The primary use case here is to redefine the SetVersion target to be something else, so it needs to come after the SetVersion target to override it.